index.html

```diff
@@ -12,34 +12,47 @@
+                        <h2>Home</h2>
+                    </a>
                 </li>
-                <li class="two-column-grid_expand-two gap-bg align-center">
-                    <img src="/icons/hashtag.png" alt="Hashtag">
-                    <h2><a href="#">Explore</a></h2>
+                <li>
+                    <a class="two-column-grid_expand-two gap-md align-center" href="#">
+                        <svg fill="currentColor" height="26px" width="26px"><g><path d="M21 7.337h-3.93l.372-4.272c.036-.412-.27-.775-.682-.812-.417-.03-.776.27-.812.683l-.383 4.4h-6.32l.37-4.27c.037-.413-.27-.776-.68-.813-.42-.03-.777.27-.813.683l-.382 4.4H3.782c-.414 0-.75.337-.75.75s.336.75.75.75H7.61l-.55 6.327H3c-.414 0-.75.336-.75.75s.336.75.75.75h3.93l-.372 4.272c-.036.412.27.775.682.812l.066.003c.385 0 .712-.295.746-.686l.383-4.4h6.32l-.37 4.27c-.036.413.27.776.682.813l.066.003c.385 0 .712-.295.746-.686l.382-4.4h3.957c.413 0 .75-.337.75-.75s-.337-.75-.75-.75H16.39l.55-6.327H21c.414 0 .75-.336.75-.75s-.336-.75-.75-.75zm-6.115 7.826h-6.32l.55-6.326h6.32l-.55 6.326z"></path></g></svg>
+                        <h2>Explore</h2></a>
                 </li>
-                <li class="two-column-grid_expand-two gap-bg align-center">
-                    <img src="/icons/bell.png" alt="Bell icon.">
-                    <h2><a href="#">Notifications</a></h2>
+                <li>
+                    <a class="two-column-grid_expand-two gap-md align-center" href="">
+                        <svg fill="currentColor" height="26px" width="26px"><g><path d="M21.697 16.468c-.02-.016-2.14-1.64-2.103-6.03.02-2.532-.812-4.782-2.347-6.335C15.872 2.71 14.01 1.94 12.005 1.93h-.013c-2.004.01-3.866.78-5.242 2.174-1.534 1.553-2.368 3.802-2.346 6.334.037 4.33-2.02 5.967-2.102 6.03-.26.193-.366.53-.265.838.102.308.39.515.712.515h4.92c.102 2.31 1.997 4.16 4.33 4.16s4.226-1.85 4.327-4.16h4.922c.322 0 .61-.206.71-.514.103-.307-.003-.645-.263-.838zM12 20.478c-1.505 0-2.73-1.177-2.828-2.658h5.656c-.1 1.48-1.323 2.66-2.828 2.66zM4.38 16.32c.74-1.132 1.548-3.028 1.524-5.896-.018-2.16.644-3.982 1.913-5.267C8.91 4.05 10.397 3.437 12 3.43c1.603.008 3.087.62 4.18 1.728 1.27 1.285 1.933 3.106 1.915 5.267-.024 2.868.785 4.765 1.525 5.896H4.38z"></path></g></svg>
```

Comment:
Steven put the svg and p (you have used h2) in their own divs whithin the a-tag. It might sort your alignment problem?

Reply:
That line was the one that was taken out, but I implemented a very similar solution with the svg and a div holding the text. The problem with the alignment is that when I increase the size of the svg, its container takes more space but the drawing stays the same. This forces sibling elements to move or the line height to increase while the icon I am trying to make bigger is not affected at all.